californium-core/src/main/java/org/eclipse/californium/core/network/UdpMatcher.java

```diff
@@ -415,6 +424,18 @@ private boolean isResponseRelatedToRequest(final Exchange exchange, final Correl
 		}
 	}
 
+	private boolean isResponseRelatedToMulticastRequest(final Exchange exchange, final CorrelationContext responseContext) {
```

Comment:
Why do we need this separate method for comparing the correlation context? Since multicast always uses plain UDP (never DTLS) the correlation context will never contain a SESSION_ID and thus the comparison is basically the same as the one in `isResponseRelatedToRequest`, right?